examples/shunit2/test_helper.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+usage() {
+  printf "Usage: "
+  for tf_var in "${vars[@]}" ; do
+    printf "$tf_var=... "
+  done
+  printf "%s\n" "$0"
+  exit 1
+}
+
+err() {
+  echo "ERROR: $*" ; exit 1
+}
+
+vars=(
+  AWS_ACCESS_KEY_ID
+  AWS_SECRET_ACCESS_KEY
```

Comment:
@aarongorka , please note that use of access keys in the tests is only temporary until I figure out a better way based on using Roles etc.